proposals/v3/api.md

```diff
@@ -536,6 +527,44 @@ Get the full set of metainfo for a specified device.
 }
 ```
 
+**Fields**
+
+| Field | Description |
+| ----- | ----------- |
+| *timestamp* | An RFC3339 timestamp describing the time that the device info was gathered. |
+| *id* | The globally unique ID for the device. |
+| *type* | The device type, as specified by the plugin. This is the last element in the namespaced device kind. |
+| *kind* | The device kind, as specified by the plugin. |
+| *metadata* | A map of arbitrary values that provide additional data for the device.. |
+| *plugin* | The ID of the plugin that manages the device. |
+| *info* | A human readable string providing identifying info about a device. |
+| *tags* | A list of the tags associated with this device. One of the tags will be the 'id' tag which should match the `id` field. |
+| *capabilities* | Specifies the actions which the device is able to perform (e.g. read, write). See below for more. |
+| *output* | A list of the output types that the device supports. |
+
+
+**Capabilities Fields**
+
+| Field | Description |
+| ----- | ----------- |
+| *mode* | A string specifying the device capabilities. This can be "ro" (read only), "rw" (read write), "wo" (write only). |
```

Comment:
If "r" and "w" are not reserved for anything then I think we can just let them be read only and write only respectively , instead of "ro" and "wo".

Reply:
true. I don't have strong opinions either way. I think both are common enough